### ISSUE TITLE
Loss doesn't seem to go down at all...

### ISSUE THREAD

**Issue by serfaniane** [crisb-DUT/DnCNN-tensorflow#2]
The loss with the code out of the box doesn't seem to want to go down at all. It fluctuates at around 5.9 and stays there without going down. Could you please tell me what this means? Thanks.

**Comment by ZhuLinweiCityU** [comment]
The loss only fluctuates instead of decreasing.

**Comment by serfaniane** [comment]
Thank you for your response. It appears that the network doesn't actually denoise images, no matter how many epochs it's seen.

**Comment by ZhuLinweiCityU** [comment]
@serfaniane The problem maybe that the weight and bias of every layer are always set as the initial values.

**Comment by SaiGokulHariharan** [comment]
Choosing a lower learning rate (.00001) and lower values for momentum (beta1 = beta2 = .05) decreases the loss (I have the feeling that it will go to a lower value, may be 3, and fluctuate there). 

Surprisingly the matlab implementation uses the default values (specified in this code) and it seems to work there (https://github.com/cszn/DnCNN/blob/master/TrainingCodes/DnCNN_TrainingCodes_v1.1/DnCNN_train.m).

**Comment by kwrgyc** [comment]
The loss doesn't decrease and the PSNR gets no promotion.

**Comment by wangWilson** [comment]
I run this network with the following values:
when lr=0.1,loss=5.9 and psnr=-4.06   4h
when lr=0.0001,loss=3.5 and psnr=-1.59    6.5h
Is there anything better than this？

**Comment by crisb-DUT** [comment]
I have fix it
